### DISCUSSION TITLE
[2022/03/15] Frontend Night

### DISCUSSION THREAD

**Discussion by jjenko** [wantedly/frontend-night#196] [Archive]
## Policy

- 資料の準備をがんばらない
- 参加・途中退席は自由
- スキップしない

話したい・聞きたいネタを書いてく ✏️

ハッシュタグ [#frontend_night](https://twitter.com/hashtag/frontend_night?src=hash)

Prev: https://github.com/wantedly/frontend-night/discussions/195 

 _Created from https://github.com/wantedly/frontend-night/discussions/179 by [issue-creator](https://github.com/rerost/issue-creator)_

**Comment by izumin5210** [comment]
[ABEMA WebにChunkの細分化を導入してパフォーマンス向上をしました \| CyberAgent Developers Blog](https://developers.cyberagent.co.jp/blog/archives/34643/)

**Reply by izumin5210** [reply]
記事からリンクされてた granular chuking の解説が良かった
[Improved Next\.js and Gatsby page load performance with granular chunking](https://web.dev/granular-chunking-nextjs/)

**Comment by izumin5210** [comment]
[Next\.jsのReact Hookへの最適化は一体何をしているのか \- tosuke](https://scrapbox.io/tosuke/Next.js%E3%81%AEReact_Hook%E3%81%B8%E3%81%AE%E6%9C%80%E9%81%A9%E5%8C%96%E3%81%AF%E4%B8%80%E4%BD%93%E4%BD%95%E3%82%92%E3%81%97%E3%81%A6%E3%81%84%E3%82%8B%E3%81%AE%E3%81%8B)

**Comment by izumin5210** [comment]
[How to Upgrade to the React 18 Release Candidate – React Blog](https://reactjs.org/blog/2022/03/08/react-18-upgrade-guide.html)
